tasks/cluster.yml

```diff
@@ -5,13 +5,16 @@
 
 - set_fact:
     cluster_userdata: "{{ lookup('template', 'lcg_userdata.j2') }}"
+  when: item_c.cluster_type == "ec2"
```

Comment:
Too many conditional (`when`) here, IMHO you could use one `block` here. What do you think?

Reply:
done!